4/john-wiegley-captions.srt

```diff
@@ -783,7 +791,7 @@ Could not match.
 
 179
 00:12:30,800 --> 00:12:36,200
-Oh! because inaudible in io.
+Oh! because [inaudible] in IO.

```

Comment:
Yeah, I can't hear it perfectly either, but `parseTest` seems like a reasonable guess given the context and what's on screen.